src/coreclr/src/jit/importer.cpp

```diff
@@ -3632,6 +3632,17 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
             op1 = impPopStack().val;
             if (opts.OptimizationEnabled())
             {
+                if (!opts.IsReadyToRun() && op1->OperIs(GT_CNS_STR))
```

Comment:
Why is this disabled for R2R?

Reply:
I might need some help because I can't figure out why crossgen2 fails with `AccessViolationExcception` on a random method (e.g. `AppDomain.get_MonitoringSurvivedProcessMemorySize`) 🙁 it sounds like I incorrectly updated the managed jit interface but I don't know what exactly...

Reply:
E.g. if I remove `getStringLength` from `ThunkInput.txt`, re-gen it using `gen.bat` -- the AVE disappears.